src/db/QDjangoQuerySet.cpp

```diff
@@ -400,7 +410,7 @@ QDjangoQuery QDjangoQuerySetPrivate::insertQuery(const QVariantMap &fields) cons
     foreach (const QString &name, fields.keys()) {
         const QDjangoMetaField field = metaModel.localField(name.toLatin1());
         fieldColumns << db.driver()->escapeIdentifier(field.column(), QSqlDriver::FieldName);
-        fieldHolders << QLatin1String("?");
+        fieldHolders << QString::fromLatin1(":%1").arg(name);
```

Comment:
Why are we touching insertQuery?

I think this change is what is causing the test build to fail